### PR TITLE
Initial google test was improved defining a beforeEach call

### PR DESCRIPTION
**Pull request by juan-carvajal**

### CODE REVIEW

File: cypress/integration/google.spec.ts [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+
+    it('should have a title',async () => {
+
+      cy.title().should('eq', 'Google')
+    });
+
+  })
+
+
+
```
Comment (yesid-lopez) [inline cypress/integration/google.spec.ts]: Delete those empty lines

Comment (juan-carvajal) [reply cypress/integration/google.spec.ts]: Done

**Review by yesid-lopez — APPROVED** [review]

File: cypress/integration/google.spec.ts [hunk 2]
```diff
@@ -1,6 +1,15 @@
-describe('This is the first example of cypress', () => {
-  it('should have a title', () => {
-    cy.visit('www.google.com')
-    cy.title().should('eq', 'Google')
-  });
+describe('Given a SDET learning cypress', () => {
+
+  describe('when open Google Page', () => {
+
+    beforeEach(async () => {
```
Comment (yesid-lopez) [inline cypress/integration/google.spec.ts:5]: delete async bc it is useless

File: cypress/integration/google.spec.ts [hunk 3]
```diff
@@ -1,6 +1,15 @@
...
+      cy.visit('www.google.com')
+    });
+
+    it('should have a title',async () => {
```
Comment (yesid-lopez) [inline cypress/integration/google.spec.ts:9]: delete async bc it is useless

File: cypress/integration/google.spec.ts [hunk 4]
```diff
@@ -1,6 +1,15 @@
...
+
```
Comment (yesid-lopez) [inline cypress/integration/google.spec.ts:10]: delete blank space

**Pull request merged**